app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -47,11 +46,11 @@
 
     private Adapter adapter;
 
-    public static ContributorsFragment newInstance() {
-        return new ContributorsFragment();
+    public ContributorsFragment() {
     }
 
-    public ContributorsFragment() {
+    public static ContributorsFragment newInstance() {
```

Comment:
Could you please align the order of static methods and member methods with other classes? Thanks.